src/lib/config.js

```diff
@@ -1,20 +1,6 @@
 'use strict'
 
 const Config = require('five-bells-shared').Config
```

Comment:
I'll do this when I need to touch Config again for adding client certs, right now this isn't a priority